frontend/src/lib/components/DateFilter/DateFilter.tsx

```diff
@@ -215,27 +214,25 @@ export function DateFilter({
         )
 
     return (
-        <LemonButtonWithDropdown
-            data-attr="date-filter"
-            id="daterange_selector"
-            onClick={isVisible ? close : open}
-            disabled={disabled}
-            disabledReason={disabledReason}
-            className={clsx('text-nowrap', className)}
-            size={size ?? 'small'}
-            type="secondary"
-            dropdown={{
-                onClickOutside: close,
-                visible: isVisible,
-                overlay: popoverOverlay,
-                placement: dropdownPlacement,
-                actionable: true,
-                closeOnClickInside: false,
-                additionalRefs: [rollingDateRangeRef, '.datefilter-datepicker'],
```

Comment:
ELI5 why don't we need the `'.datefilter-datepicker'` ref now?

Reply:
Ah `* Works also with strings, matching classnames or ids, for antd legacy components that don't support refs` and there aren't any any more 🚀